src/test/java/uk/nhs/digital/nhsconnect/nhais/service/FhirToEdifactServiceTest.java

```diff
@@ -83,44 +104,85 @@ public void when_convertedSuccessfully_dependenciesCalledCorrectly() throws Exce
     }
 
     @Test
-    public void when_convertedSuccessfully_edifactIsCorrect() throws Exception {
-        Patient patient = createPatient();
+    public void when_convertedSuccessfully_edifactIsCorrect() {
+        Parameters parameters = createParameters();
 
-        TranslatedInterchange translatedInterchange = fhirToEdifactService.convertToEdifact(patient, ReferenceTransactionType.TransactionType.ACCEPTANCE);
+        TranslatedInterchange translatedInterchange = fhirToEdifactService.convertToEdifact(parameters, ReferenceTransactionType.TransactionType.ACCEPTANCE);
 
         String expected = "UNB+UNOA:2+GP123+HA456+200427:1737+00000045'\n" +
-                "UNH+00000056+FHSREG:0:1:FH:FHS001'\n" +
-                "BGM+++507'\n" +
-                "NAD+FHS+HA456:954'\n" +
-                "DTM+137:202004271737:203'\n" +
-                "RFF+950:G1'\n" +
-                "S01+1'\n" +
-                "RFF+TN:5174'\n" +
-                "UNT+8+00000056'\n" +
-                "UNZ+1+00000045'";
+            "UNH+00000056+FHSREG:0:1:FH:FHS001'\n" +
```

Comment:
This is not a valid EDIFACT file. Please use the Appendix J examples as the basis for tests.

Reply:
will fix-forward using new integration test framework